### PR TITLE
[CSFS-5] Create & Style the Customer Signup Page 

### PR DESCRIPTION
**Pull request by Oge-nna**
## Task: `[CSFS-5] Create & Style the Customer Signup Page`
- `Created and styled a responsive sign up page.`
SignUp
SignUp2
SignUpMobile
SignUpMobile2
SignUpTab
SignUpTab2

### CODE REVIEW

File: submissions/ogennaOgbogu/htmlForms/signUp.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                                            <input type="text" name="newsletter" id="newsletter" placeholder="Enter your email address" required/>
+                                        <button type="button" id="submitbutton">SUBMIT</button>
+                                        </div>
+                                        
+                                </div> 
+                            </div>      
+
+                            <div class="social-network">
+                                <p>CONNECT WITH US</p>
+                                <div class="social-img">
```
Comment (ToluAkin) [inline submissions/ogennaOgbogu/htmlForms/signUp.html:144]: @Oge-nna Style the images in your css

File: submissions/ogennaOgbogu/htmlForms/signUp.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+                                    <img src="..//htmlForms/Images/facebook logo.png" width="40px" height="30px">
+                                    <img src="..//htmlForms/Images/gmail logo.png" width="40px" height="30px">
+                                    <img src="..//htmlForms/Images/twiiter logo.png" width="40px" height="30px">
+                                    <img src="..//htmlForms/Images/youtube logo.png" width="40px" height="30px">
+                                </div>
+                            </div>
+                        </div>
+        </footer> 
+        <div class="lcright">
```
Comment (ToluAkin) [inline submissions/ogennaOgbogu/htmlForms/signUp.html:153]: use descriptive class names @Oge-nna

File: submissions/ogennaOgbogu/htmlForms/signUp.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        <footer class="footer">
+                        <div class="aboutus">
+                            <p><a href="#">Contact Us</a></p>
+                            <p><a href="#">About Us</a></p>
+                            <p><a href="#">Terms and Conditions</a></p>
+                        </div>     
+
+                        <div class="rightfooter">
+
+                            <div class="candc">
```
Comment (ToluAkin) [inline submissions/ogennaOgbogu/htmlForms/signUp.html:129]: use descriptive class names @Oge-nna

File: submissions/ogennaOgbogu/htmlForms/signUp.html [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+                        </div>
+                        </div>
+                    </form>
+                </div>   
+        </div>           
+            
+
+            <!--footer-->
+        <footer class="footer">
+                        <div class="aboutus">
```
Comment (ToluAkin) [inline submissions/ogennaOgbogu/htmlForms/signUp.html:121]: Separate the class name with `-` (dash) for readability @Oge-nna

File: submissions/ogennaOgbogu/htmlForms/signUp.html [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+                            <div>
+                                <input type="checkbox" name="terms" id="terms"/>
+                                <label for="terms">I agree to the Terms and Conditions</label>
+                            </div>
+                        </div>
+                        
+                        <br>
+
+                        <div class="button1">
+                        <button type="button" id="signupbutton">SIGN UP</button>
```
Comment (ToluAkin) [inline submissions/ogennaOgbogu/htmlForms/signUp.html:111]: Separate the ID with `-` (dash) for readability @Oge-nna

File: submissions/ogennaOgbogu/htmlForms/signUp.html [hunk 6]
```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Sign Up</title>
+        <link rel="stylesheet" type="text/css" href="..//CSS/signupStylesheet.css">
```
Comment (ToluAkin) [inline submissions/ogennaOgbogu/htmlForms/signUp.html:8]: Are you sure this connect is correct? @Oge-nna

File: submissions/ogennaOgbogu/htmlForms/signUp.html [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+                        customers.
+                        </p>
+                    <br>
+                    <br>
+                    <p>Already got an account? <strong><a href="#">Sign in</a></strong></p>
+                </div>  
+            </div> 
+            <!--right side-->    
+                <div class="form">
+                    <form action="#" method="GET">
```
Comment (ToluAkin) [inline submissions/ogennaOgbogu/htmlForms/signUp.html:50]: `action` and `method` attributes are not needed. @Oge-nna

File: submissions/ogennaOgbogu/htmlForms/signUp.html [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+                        </div>
+                        
+                        <div class="rightside">
+                            <label for="password2">Confirm Password:</label>
+                            <br>
+                            <input type="password2" class="text" name="password2" id="password2"/>
+                            <br>
+                        </div>
+                        
+                        <div class="tandc">
```
Comment (ToluAkin) [inline submissions/ogennaOgbogu/htmlForms/signUp.html:101]: use descriptive class names @Oge-nna

**Review by ToluAkin — CHANGES_REQUESTED** [review]
The form labels for and input field names is not descriptive. @Oge-nna Good job done so far

File: submissions/ogennaOgbogu/htmlForms/signUp.html (hunk 6 above)
Comment (Oge-nna) [reply submissions/ogennaOgbogu/htmlForms/signUp.html:8]: Hi @ToluAkin, I don't understand your question, kindly explain.

File: submissions/ogennaOgbogu/htmlForms/signUp.html (hunk 1 above)
Comment (Oge-nna) [reply submissions/ogennaOgbogu/htmlForms/signUp.html:144]: Hi @ToluAkin, I also don't understand, style in what way? Nest them in hyperlinks or? Please explain.

File: submissions/ogennaOgbogu/htmlForms/signUp.html (hunk 6 above)
Comment (ToluAkin) [reply submissions/ogennaOgbogu/htmlForms/signUp.html:8]: I mean the path to the css file @Oge-nna . There is a double slash.

File: submissions/ogennaOgbogu/htmlForms/signUp.html (hunk 1 above)
Comment (Oge-nna) [reply submissions/ogennaOgbogu/htmlForms/signUp.html:144]: Oh. Nvm. I understand now. Thank you.

**Pull request closed without merging**